### DISCUSSION TITLE
Consider change padding-top in Sample for Bootstrap fixed-top navbar

### DISCUSSION THREAD

**Discussion by surrealist** [Dotnet-Boxed/Templates#33] [Q&A]
Please consider change `<body>` padding-top from **50px** to **70px** in the Sample projects

Bootstrap 3 document states this in section **Fixed to Top**
http://getbootstrap.com/components/#navbar-fixed-top

> **Body padding required**
> The fixed navbar will overlay your other content, unless you add padding to the top of the <body>. Try out your own values or use our snippet below. Tip: By default, the navbar is 50px high.
> 
> ``` css
> body { padding-top: 70px; }
> ```

I found **padding-top: 50px** in the following places:
- Boilerplate.Web.Mvc5.Sample (`Content/Site.css` and `Content/Site.less`)
- Boilerplate.Web.Mvc6.Sample (`Styles/site.scss`)

I think this is not actually related to core of the boilerplate. But when apply correct padding-top as suggested by Bootstrap, it will has space between navbar and jumbotron (or other page contents) and that _make it really looks better_.

// As everyone knew, default ASP.NET MVC template from Microsoft uses **50px** too, but this boilerplate aims to be better than the default template, doesn't it? :smirk: 

Please tell me if I should send send you a pull request.

**Comment by RehanSaeed** [comment]
Thanks! Yes we want this to be better than the Microsoft template by far! I'm in the middle of a major update and I will merge your pull request as soon as I can.

**Comment by RehanSaeed** [comment]
Ok, I've now given this a go and I can see why they chose 50px. It's so that the carousel touches the bottom of the nav bar. The docs you linked says "Try out your own values", hence the 50px.
